server/controllers/finance/cash.js

```diff
@@ -154,11 +154,9 @@ function listPayment(options) {
   `;
 
   filters.period('period', 'date');
-  filters.dateFrom('custion_period_start', 'date');
+  filters.dateFrom('custom_period_start', 'date');
```

Comment:
👍  Good catch! FYI This was addressed in a very small PR here https://github.com/IMA-WorldHealth/bhima-2.X/pull/1774